swerve100/src/main/java/frc/robot/subsystems/SwerveDriveSubsystem.java

```diff
@@ -122,11 +122,11 @@ public SwerveDriveSubsystem(DriverStation.Alliance alliance, double currentLimit
             case COMP_BOT:
                 headingController = new ProfiledPIDController( //
                         1, // kP
-                        .5, // kI
-                        0.15, // kD
+                        0, // kI
```

Comment:
kinda surprised that zero I and D really works very well.  could you say more about this?

Reply:
also were you using the comp bot?   i thought you were using swerve 2?